cmd/get.go

```diff
@@ -52,6 +53,41 @@ func buildAlias(name string) []string {
 	return []string{strings.ToLower(name), removeTrailingSIfAny(strings.ToLower(name)), removeTrailingSIfAny(name)}
 }
 
+func printResource(result interface{}, format string) error {
+	if format == "json" {
+		jsonOutput, err := json.MarshalIndent(result, "", "  ")
+		if err != nil {
+			return fmt.Errorf("error marshalling JSON: %s\n%s", err, result)
+		}
+		fmt.Println(string(jsonOutput))
+	} else if format == "name" {
+		// show Kind/Name
+		switch res := result.(type) {
+		case []resource.Resource:
+			for _, r := range res {
+				fmt.Println(r.Kind + "/" + r.Name)
+			}
+		case resource.Resource:
+			fmt.Println(res.Kind + "/" + res.Name)
+		default:
+			return fmt.Errorf("unexpected resource type")
+		}
+	} else if format == "yaml" {
+		switch res := result.(type) {
+		case []resource.Resource:
+			for _, r := range res {
+				fmt.Println("---") // '---' indicates the start of a new document in YAML
+				r.PrintPreservingOriginalFieldOrder()
+			}
+		case resource.Resource:
+			res.PrintPreservingOriginalFieldOrder()
+		default:
+			return fmt.Errorf("unexpected resource type")
+		}
+	}
```

Comment:
else {
  panic

Reply:
Or return an error

Reply:
i prefer to return the error and let the caller deal with it